tests/controllers/openAi.test.ts

```diff
@@ -30,30 +31,28 @@ jest.mock('openai', () => {
 
 describe('processAnalysis', () => {
   it('should respond with a 400 if content is missing', async () => {
-    await request(app).post('/api/openai')
-      .send({ promptType: 'analyze'})
-      .expect(400)
-      .expect({ error: 'Both content and promptType are required fields' })
+    const { status, body } = await request(app).post('/api/openai').send({ promptType: 'analyze' })
```

Comment:
Rather than using supertest's way of testing, switch to the standard which offers better stack traces and also doesn't bother eslint